src/SFA.DAS.LevyTransferMatching/Application/Commands/SetApplicationAcceptance/SetApplicationAcceptanceCommandHandler.cs

```diff
@@ -22,11 +23,18 @@ public SetApplicationAcceptanceCommandHandler(ILogger<SetApplicationAcceptanceCo
 
         public async Task<bool> Handle(SetApplicationAcceptanceCommand request, CancellationToken cancellationToken)
         {
-            if (request.Acceptance != Types.ApplicationAcceptance.Accept)
-            {
-                throw new NotImplementedException();
-            }
+            HttpStatusCode httpStatusCode;
+
+            if (request.Acceptance == Types.ApplicationAcceptance.Accept)
+                httpStatusCode = await AcceptFunding(request, cancellationToken);
+            else
+                httpStatusCode = await DeclineFunding(request);
 
+            return httpStatusCode == HttpStatusCode.NoContent;
```

Comment:
I'd have thought checking for any success code would be sufficient, rather than being this specific. Why would we want an error here if a change were made on the endpoint to return content?

I'm also not convinced about returning a bool here vs a specific type vs no return at all, and allowing any exception to bubble up, but it looks like this was part of the original version and not yours, so we can look at it with a tech debt task.

Reply:
OK, happy to review this as a tech debt task.